src/app/login/Login.types.ts

```diff
@@ -0,0 +1,8 @@
+import { UseMutationResult } from 'react-query';
+
+import { LoginPayload } from 'api/types';
+
+export type LoginProps = Pick<
```

Comment:
Myślę że można każdy z tych elementów wpisać osobno, bez użycia UseMutationResult. Z jednego prostego względu - to jest nieczytelne, kiedy cztery z typów są `unknown` :D
I druga sprawa - jak się powinien nazywać poprawnie props `mutate`? ;)
I jeszcze trzecia - a jak się może nazywać inaczej `data`? Tak bardziej opisowo, co to jest?